Script.md

```diff
@@ -0,0 +1,25 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.
+Lodgel is designed to take the hassle out of out of your property management experience.
+You can quickly delegate tasks to your turnover assistants, conveniently schedule guests and securely capture 
+their payments at the click of a button, all while enjoying the support of our great team.
+
+Users register for an account, and right away, they can start adding properties to their list.
+To ensure each guest is taken care of, checklists can be made that detail tasks that need to be taken care of
+before, during, and after each visit. 
+
+Turnover assistants are an essential element of every property management business. Each property has 
+a turnover assistant assigned to take care of it by default, but they can be moved around if necessary. 
+Each turnover assistant has a map attached to their profile, which conveniently shows their location relative to 
+the property they maintain. You can also assign them to properties if needed, which saves you time 
+and effort in finding someone to cover another turnover assistant in the event they are unavailable.
+
+You can also keep tabs on your guest throughout their entire rental experience. After putting in a reservation,
+you can keep your turnover assistants up to date with your guests needs by adding items to their checklists. Since your 

```

Comment:
Are we keeping tabs on our guest or our guests? Also, the next sentence contains a lot of people and pronouns so it could be edited to make it more clear about who is doing what when. Maybe try "after you enter a reservation, you can keep your turnover assistants updated with your guest's needs by adding items to the assistant's checklists."

Reply:
It's more the manager of the properties that is keeping up on the guests. I fixed up the pronouns, hopefully a bit better now.